mne_bids_pipeline/steps/preprocessing/_08a_apply_ica.py

```diff
@@ -224,7 +224,7 @@ def apply_ica_raw(
     raw_fname = in_files.pop(in_key)
     assert len(in_files) == 0, in_files
     out_files = dict()
-    out_files[in_key] = raw_fname.copy().update(processing="clean")
+    out_files[in_key] = raw_fname.copy().update(processing="clean", split=None)
```

Comment:
Should we check **all** code that generates output filenames to ensure we set `split=None` there?

Reply:
I think we could do it in the function that prepares the output filenames actually. I'll give it a shot

Reply:
Should be working now!